TRANSLATIONS.rst

```diff
@@ -0,0 +1,145 @@
+🐍 🌍 🚀 Translation Release Strategy
+======================================
+
+Thanks for your interest in translating the micro:bit Python Editor.
+Translations for all the Micro:bit Educational Foundation’s editors and
+apps are contributed by an amazing team of volunteers and we’d love for
+you to be a part of this too.
+
+This document sets out the strategy and process for how the Foundation
+manages the translation of the Python Editor hosted at
+python.microbit.org
+
+1. Translation and Management in the Python Editor
+--------------------------------------------------
+
+Translation is the act of taking the written materials in one language
+and converting them into another language in the most meaningful way
+possible. In terms of the micro:bit Python Editor, translation happens
+on the `strings marked for translation in the projects'
+codebase <https://github.com/bbcmicrobit/PythonEditor/lang/en.js>`__. The default language of
+the editor is English and you may select another available language
+using the Globe icon in the main menu.
+
+1.1 Crowdin
++++++++++++
+The Micro:bit Educational Foundation use the `Crowdin
+platform <https://crowdin.com/project/microbitorg>`__ to manage
+translations to manage and support community translations for it’s web
+projects. We have an amazing team of volunteer translators and
+proofreaders in our global micro:bit community and Crowdin helps us
+support them through the process. There is some more information on how
+this works at
+`microbit.org/translate <https://microbit.org/translate/>`__
+
+1.2 Translating the editor
+++++++++++++++++++++++++++
+We encourage translators to use
+`Crowdin <https://crowdin.com/project/microbitorg>`__ as it has a
+rigorous process for translation, proofreading and syncronisation and
+the Foundation use it to communicate and help guide the translation
+community.
+
+In the microbit.org project, you’ll find a `folder titled apps that
+contains a sub folder for the Python
+Editor. <https://crowdin.com/project/microbitorg/ar#/new/apps/python-editor>`__
+This folder contains the current English language translation
+**en.json** file with the strings required for translation. Selecting
+this file opens the Crowdin editor where you can submit your own or use
+a suggested translation for the string. It will then need to be verified
+by a proofreader for that language.
+
+1.3 Syncing Translations
+++++++++++++++++++++++++
+Crowdin can perform a sync between itself and Github, meaning that we can pass language updates either way. This
+is currently triggered manually. When a new release of the editor is
```

Comment:
Sounds like this also takes into account beta release?

If that's the case then each beta release would be deployed without translations for the new strings, and the translation would only be deploted by the next beta release?
I think this is fine, but it does mean there is a delay between releasing a feature and having the translations available.

Also, if there were any primary releases that bypass the beta release, we might need to manage that differently, but we can cross that bridge when we come it it.

Reply:
I think this is okay if I understand your concern correctly. eg
1. new beta feature added and strings updated in en.js
2. beta feature released with new strings only in english
3. Translation work can happen on the new strings from this point

The feature can be released with partial/full translation in available languages

Perhaps we should note that features in beta are experimental, so translation efforts should be focussed on the live editor?

Reply:
Yeah, I wrote this before I read the following sections, which I interpreted as "the strings will only be updated into Crowdin once they are officially "frozen" (likely at the first RC release)".
I left the comment because I am not quite sure when we will be updating the strings. Doing it only at RC is fine, but it means that any new/updated strings in the Beta Editor will  always be available in english only, as they will only be frozen when an RC is ready, and at that point the release into the main editor will be imminent.

Reply:
I think this comment and https://github.com/bbcmicrobit/PythonEditor/pull/234/files/d14b75aefe8eb5a80b4cbf4aeff04e7660d9fa1f#diff-62acef342617ece5a91fe47442d38d91 still need to implemented?